src/Components/Shoppinglist.css

```diff
@@ -1,8 +1,12 @@
-.shopping{
-    width:100%;
-    height:100vh;
+.shopping {
+  width: 100%;
+  height: 100vh;
```

Comment:
This should not be height but min-height. If you specify height the scroll will not work